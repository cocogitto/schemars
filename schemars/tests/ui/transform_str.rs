use cocogitto_schemars::JsonSchema;

#[derive(JsonSchema)]
#[cocogitto_schemars(transform = "x")]
pub struct Struct;

fn main() {}
