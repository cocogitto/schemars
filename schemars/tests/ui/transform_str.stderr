error: Expected a `fn(&mut Schema)` or other value implementing `schemars::transform::Transform`, found `&str`.
       Did you mean `#[cocogitto_schemars(transform = x)]`?
 --> tests/ui/transform_str.rs:4:24
  |
4 | #[cocogitto_schemars(transform = "x")]
  |                        ^^^
